test-prow-e2e.sh

```diff
@@ -32,5 +32,59 @@ if [ ! -d node_modules ]; then
   yarn install
```

Comment:
I don't see anything Prow-specific, why is `prow` in script name?

Reply:
I'll check it